app/components/Form/ProjectFundingAgreementForm.tsx

```diff
@@ -119,13 +121,28 @@ const ProjectFundingAgreementForm: React.FC<Props> = (props) => {
     handleChange(fundingAgreement.newFormData, "staged");
   };
 
+  const router = useRouter();
+  const navigateToReviewAndSubmit = () => {
+    console.log("navigateToReviewAndSubmit");
+    router.push(getProjectRevisionPageRoute(projectRevision.id.toString()));
+  };
+
   return (
     <>
       {projectRevision.projectFundingAgreementFormChanges.edges.length ===
         0 && (
-        <Button onClick={addFundingAgreement} style={{ marginRight: "1rem" }}>
-          Add Funding Agreement
-        </Button>
+        <>
+          <h3>Is this a funded project?</h3>
+          <Button onClick={addFundingAgreement} style={{ marginRight: "1rem" }}>
+            Yes
+          </Button>
+          <Button
+            onClick={navigateToReviewAndSubmit}
+            style={{ marginRight: "1rem" }}
+          >
+            No
+          </Button>
```

Comment:
These should be radio buttons, conditionally rendering the form whether or not 'yes' is selected or not
@nanyangpro will provide a very quick wireframe today